spec/suma/i18n_spec.rb

```diff
@@ -58,6 +58,36 @@
     end
   end
 
+  describe "ensure_english_interpolation_values" do
```

Comment:
These tests need to go 'through the front door' and use `import_csv`. If that's hard, you can refactor `import_csv` to pull out a high-level, easier-to-test method.

Try to avoid testing implementation details like this explicitly. In some cases, it's fine for edge cases that are hard to exercise through the front door. But in general, you want to avoid tests that break for internal refactorings (and this is definitely an internal thing, no one would need to call it directly). For one benefit- you don't ever test this code is actually used :)